src/map.cpp

```diff
@@ -216,10 +216,15 @@ void Map::setNode(v3s16 p, MapNode & n)
 	// Never allow placing CONTENT_IGNORE, it fucks up stuff
 	if(n.getContent() == CONTENT_IGNORE){
 		bool temp_bool;
-		errorstream<<"Map::setNode(): Not allowing to place CONTENT_IGNORE"
-				<<" while trying to replace \""
-				<<m_nodedef->get(block->getNodeNoCheck(relpos, &temp_bool)).name
-				<<"\" at "<<PP(p)<<" (block "<<PP(blockpos)<<")"<<std::endl;
+		errorstream << "Map::setNode(): Not allowing to place CONTENT_IGNORE"
+			<< " while trying to replace \""
+			<< m_nodedef->get(block->getNodeNoCheck(relpos, &temp_bool)).name
+			<< "\" at " << PP(p) << " (block " << PP(blockpos) << ") "
+			<< "with an unrecognized node: (" << " Param0=" << n.getContent()
+			<< ", param1= " << unsigned(n.getParam1())
```

Comment:
printing the params doesn't really make any sense here, it will almost always be `param0=<what ID ignore has> param1=0 param2=0`

Reply:
so nix the params altogether? or keep param1? or?  example of not all 0's https://imgur.com/a/9L0kk

Wouldn't it be param0 = whatever id the mod tried to place? or is it somewhere converted to a ignore node id?

Reply:
Agreed all params are useless here.

Reply:
`Param0=" << n.getContent()`
but above you can see
`if(n.getContent() == CONTENT_IGNORE`